aea/runtime.py

```diff
@@ -152,9 +154,6 @@ def _start(self) -> None:
 
         Start runtime asynchonously in own event loop.
         """
-        if self._state.get() is RuntimeStates.started:
-            raise ValueError("Runtime already started!")
```

Comment:
why removed?

Reply:
cause the same check is performed in caller method. double check.